app/sections/view-private-cloud/macro.test.js

```diff
@@ -0,0 +1,158 @@
+      params: {
+        section: {
+          answers: {
+            'hosting-model': 'Our managed data center is hosted in two separate geographical locations, they both comply to the highest standards to ensure that even if one of our data centers suffers an outage, we can ensure that your daily activities are not interrupted. We also create a back up of all of our data every evening and store it separately, so in the result of any catastrophic failure, we can ensure that patient confidential information is kept secure.',
+          },
+        },
+      },
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        const hostingModelQuestionRow = $('[data-test-id="view-section-table-row-hosting-model"]');
+        const hostingModelInnerComponent = hostingModelQuestionRow
+          .find('div[data-test-id="view-section-table-row-component"]')
+          .find('[data-test-id="view-question-data-text-hosting-model"]');
+
+        expect(hostingModelQuestionRow.length).toEqual(1);
+        expect(hostingModelQuestionRow
+          .find('div[data-test-id="view-section-table-row-title"]').text().trim()).toEqual('Data center hosting model');
+        expect(hostingModelInnerComponent.length).toEqual(1);
+        expect(hostingModelInnerComponent.text().trim()).toEqual('Our managed data center is hosted in two separate geographical locations, they both comply to the highest standards to ensure that even if one of our data centers suffers an outage, we can ensure that your daily activities are not interrupted. We also create a back up of all of our data every evening and store it separately, so in the result of any catastrophic failure, we can ensure that patient confidential information is kept secure.');
+
+        done();
+      });
+  });
+
+  it('should render the requires HSCN answer', (done) => {
+    const context = {
+      params: {
+        section: {
+          answers: {
+            'requires-hscn': 'This Solution requires a HSCN/N3 connection',
+          },
+        },
+      },
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        const requiresHSCNQuestionRow = $('[data-test-id="view-section-table-row-requires-hscn"]');
+        const requiresHSCNInnerComponent = requiresHSCNQuestionRow
+          .find('div[data-test-id="view-section-table-row-component"]')
+          .find('[data-test-id="view-question-data-text-requires-hscn"]');
+
+        expect(requiresHSCNInnerComponent.length).toEqual(1);
+        expect(requiresHSCNInnerComponent.text().trim()).toEqual('This Solution requires a HSCN/N3 connection');
+
+        done();
+      });
+  });
+
+  it('should not render the requires HSCN answer if not given', (done) => {
+    const context = {
```

Comment:
I think you also need to add tests for when the model, link and summary are not provided as all fields are optional

Reply:
agree with Katie. We should test for these possibilities too